core/error-handler.ts

```diff
@@ -0,0 +1,6 @@
+// import { toast } from 'react-toastify';
+
+export default (error: string) => {
+  console.error("Error occurred:", error);
+  // toast.error(error.message || "An unexpected error occurred");
```

Comment:
Sentry will go here I guess!

Reply:
I've implemented react hot toast in the register-login-page branch. We can change though.